scrape.rb

```diff
@@ -14,9 +14,9 @@
       location = tournament.css('a').children.text
       # May want to use this date method below to get the tournament's start date
       # So that we can sort upcoming events by start date
-      # month_and_day = tournament.css('td')[-2].children.text[/^\d*\/\d*/]
-      # year = tournament.css('td')[-2].children.text[/\d\d\d\d/] || Date.today.year
-      # sortable_date = Date.parse("#{year}/#{month_and_day}")
+      month_and_day = tournament.css('td')[-2].children.text[/^\d*\/\d*/]
```

Comment:
If scrape.rb is no longer needed, we should delete it from the repo

Reply:
I think it would be useful to leave it in, just cause I will need to test more scraping to get all players for a tournament